crates/subspace-core-primitives/src/lib.rs

```diff
@@ -244,3 +251,16 @@ impl RootBlock {
         }
     }
 }
+
+/// Encoded block with mapping of objects that it contains
+#[derive(Clone)]
+#[cfg_attr(feature = "std", derive(Debug))]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "std", serde(rename_all = "camelCase"))]
+pub struct EncodedBlockWithObjectMapping {
```

Comment:
This is not a core data type for the protocol, this data structure is only used for RPC purposes and never anywhere else. This is why I didn't put it into core primitives and left duplicated, I didn't know where to put it nicely, I almost feel we need a separate crate for RPC-related types.